### DISCUSSION TITLE
verification cases results ver-1a and ver-1b

### DISCUSSION THREAD

**Discussion by federicohattab** [idaholab/TMAP8#45] [Q&A] [answered]
Hello, 

I am a new user and I am currently studying the verification cases to get a hang of the program. Everything is working well except I cannot reproduce the results shown on the website of case ver-1a (fractional release with time) and case ver-1b (normalized species concentration with time). 

This figure is the result I obtain by executing the inputs in the folder  TMAP8/test/tests/ver-1a, without any modification.
![ver-1a_comparison](https://user-images.githubusercontent.com/124260728/218057784-64eb2eb7-aeb1-4809-8d77-90f1aa317d3b.png)

These two figures concern the case ver-1b. The first is the outputs of the python comparison script, while in the second I compared the analytical solution, the u_vs_x.csv file in the gold folder and the output of the .i file run.
![ver-1b_comparison_dist](https://user-images.githubusercontent.com/124260728/218057812-6b774ba7-93bc-40b2-83ac-4046b7bfd01c.png)
![ver-1b_comparison_i_gold_analytical](https://user-images.githubusercontent.com/124260728/218077312-c0feba84-dd18-4dc0-a3c5-0c9823ec94ae.png)
 
any help would be fantastic, thank you.

**Comment by singhgp4321** [comment, marked as answer]
The comparison plots are for the runs with refined mesh or shorter time steps. If you refine the mesh or decrease the step time or do both, depending on which case your are running, you would be able to reproduce the results. Let me know if you need any help with that.

**Reply by federicohattab** [reply]
Thank you everything works great. I wrongly assumed that the plots were for the test runs as they are in the repository.

**Comment by simopier** [comment]
These issues have been fixed now with https://github.com/idaholab/TMAP8/pull/101 and https://github.com/idaholab/TMAP8/pull/86.
The input file available to users reproduces the results shown in the documentation.
